.babelrc

```diff
@@ -5,7 +5,13 @@
       "presets": [["env"], "react"]
     },
     "production": {
-      "plugins": ["external-helpers"]
+      "plugins": [
+        "external-helpers",
+        [
+          "transform-react-remove-prop-types",
+          { "mode": "remove", "removeImport": true }
```

Comment:
We are definitely safe to "remove" here. No use cases for needing to keep the PropTypes around 👍